EIPS/eip-5375.md

```diff
@@ -0,0 +1,293 @@
+---
+eip: 5375
+title: NFT Authorship
+description: An extension of EIP-721 for NFT authorship and author consent.
+author: Samuele Marro (@samuelemarro), Luca Donno (@lucadonnoh)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
Not sure if this should be ERC or not. I'll defer.

Reply:
The reasoning behind the ERC denomination is that it mostly concerns application-level standards. The closest EIPs ([EIP-1812](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1812.md), [EIP-721](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md) and [EIP-1155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1155.md) are all ERCs.